c/weld.h

```diff
@@ -9,6 +9,18 @@
 
 #include <stdint.h>
 
+// Log level constants -- should match WeldLogLevel.
+
+/** A Weld log level. */
+typedef uint64_t weld_log_level_t;
+
+weld_log_level_t WELD_LOG_LEVEL_OFF = 0;
```

Comment:
I wasn't sure the enum would be guaranteed to be 64-bit. How can you force it to be that?